deploy/base/profiles/bpf-recorder.json

```diff
@@ -87,6 +137,7 @@
         "setuid",
         "sigaltstack",
         "socket",
+        "stat",
```

Comment:
We need this to work consistently on Power and S390x. Without stat, we were unable to work.